### PR TITLE
Cluster history API endpoint

### PR DESCRIPTION
**Pull request by ikitiki**
introduce ``/clusters/<team>/<cluster>/history/`` endpoint to see last ``[cluster_history_entries]`` diff entries of the cluster spec changes

### CODE REVIEW

File: manifests/configmap.yaml [hunk 1]
```diff
@@ -32,4 +32,5 @@ data:
   enable_load_balancer: "true"
   api_port: "8080"
   ring_log_lines: "100"
+  cluster_history_entries: "1000"
```
Comment (ikitiki) [inline manifests/configmap.yaml]: better name?

Comment (Jan-M) [reply manifests/configmap.yaml]: cluster_manifest_history_entries

Comment (ikitiki) [reply manifests/configmap.yaml]: too long

**Comment by Jan-M** [comment]
:+1:

**Comment by ikitiki** [comment]
👍

**Pull request merged**